lib/vehicle_factory.rb

```diff
@@ -0,0 +1,20 @@
+class VehicleFactory
+  def initialize
+    
+  end
+
+  def create_vehicles(registrations)
+    created_vehicles = registrations.map do |car|
```

Comment:
Unlike `each` (which returns the original collection), `map` will return the new collection so you don't need to set this variable here or explicitly return it on line 18. 
```suggestion
    registrations.map do |car|
```